src/handlers.rs

```diff
@@ -187,7 +197,32 @@ pub struct BsoParams {
 
 #[derive(Deserialize, Serialize)]
 pub struct BsoBody {
-    pub sortindex: Option<i64>,
+    pub sortindex: Option<i32>,
     pub payload: Option<String>,
-    pub ttl: Option<i64>,
+    pub ttl: Option<u32>,
 }
+
+pub fn get_configuration(_state: State<ServerState>) -> FutureResponse<HttpResponse> {
+    // TODO: populate from static config?
+    Box::new(future::result(Ok(
+        HttpResponse::Ok().json(Configuration::default())
+    )))
```

Comment:
I'm assuming this endpoint has no need to hit the db, but let me know if that's wrong. (/cc @rfk)

Reply:
That's correct, it should be able to pull everything it needs from static app config.